.tool-versions

```diff
@@ -1 +1,2 @@
 java corretto-21.0.5.11.1
+node 20.11.0
```

Comment:
Moved the node version in .nvmrc file from the `cdk` folder to here instead to centralize tool version management